packages/amplication-client/src/Entity/ImportPrismaSchema/EntitiesImport.tsx

```diff
@@ -70,12 +74,6 @@ const EntitiesImport: React.FC<Props> = ({ match, innerRoutes }) => {
   const { resource: resourceId } = match.params;
   const { trackEvent } = useTracking();
 
-  const { stigg } = useStiggContext();
```

Comment:
What happened to the enforcement of the import prisma schema entitlement?

Reply:
`canImportDBSchema` was used for showing the beta feature UI.
The "Upload Prisma schema" button on the entity list page is wrapped by the `FeatureIndicatorContainer`